templates/includes/sidebar.html

```diff
@@ -1,4 +1,16 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a></li>
+    <form action="{% url 'logout' %}" method="post">
```

Comment:
The logout form is correctly set up with a POST method and CSRF token. Ensure that the 'logout' URL is included in your URL configurations to handle this request.